topology/src/main/java/io/camunda/zeebe/topology/state/ClusterTopology.java

```diff
@@ -195,4 +195,11 @@ public int clusterSize() {
                         && entry.getValue().state() != State.UNINITIALIZED)
             .count();
   }
+
+  public int getPartitionCount() {
```

Comment:
```suggestion
  public int partitionCount() {
```

to match naming of `clusterSize`